crates/core/src/lib.rs

```diff
@@ -205,6 +205,15 @@ impl<T: Send + OutboundWasiHttpHandler> WasiHttpView for Data<T> {
     where
         Self: Sized,
     {
+        let current_span = tracing::Span::current();
+        let uri = request.request.uri();
+        if let Some(authority) = uri.authority() {
+            current_span.record("server.address", authority.host());
+            if let Some(port) = authority.port() {
+                current_span.record("server.port", port.as_u16());
+            }
+        }
```

Comment:
Could the two chunks of code in this PR move into the same function?

Reply:
Not sure what you mean by that

Reply:
This PR currently adds telemetry code to both this function in `spin-core` and the function in `spin-trigger-http`. Could the `http.response.status_code` code instead be up here in this function?